### PR TITLE
deletePoll

### PR DESCRIPTION
**Pull request by imsandip2085**

### CODE REVIEW

File: polling-app/src/Components/UpdatePoll/UpdatePoll.js [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        </Navbar>
+        <div className="login">
+          <h1>Update Poll</h1>
+        </div>
+        {this.props.addPollStatus.response &&
+          this.props.addPollStatus.response.length &&
+          this.props.addPollStatus.response.map((val, index) => {
+            return (
+              <Card
+                style={{ width: "40rem", margin: "40px 120px" }}
```
Comment (nodeexcel) [inline polling-app/src/Components/UpdatePoll/UpdatePoll.js]: always create seperate css file

File: polling-app/src/Redux/Action/UpdatePoll/addNewOption.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      
+     let error;
+      dispatch(AddNewOptionRequest({ isLoading: true }));
+      let data = await apiRequest(
+        `https://secure-refuge-14993.herokuapp.com/add_new_option?id=${id}&option_text=${newOption}`,
+        "post",
+        {},
+        {}
+      ).then(res=>{console.log(res,"fffff")}); 
+     console.log(data.option_text,"dfhgjkjkjhhcgfhvjb");
```
Comment (nodeexcel) [inline polling-app/src/Redux/Action/UpdatePoll/addNewOption.js]: remove console

File: polling-app/src/Redux/Action/UpdatePoll/deleteOption.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import {DeleteOptionRequest,DeleteOptionError,DeleteOptionSuccess } from "../index";
+import apiRequest from "../../../service/apicall";
+
+export function deleteOptionForm(id,newOption) {
+  return async function(dispatch, getState) {  
+    try {
+      console.log(newOption,"xsdfghjkgfds");
```
Comment (nodeexcel) [inline polling-app/src/Redux/Action/UpdatePoll/deleteOption.js]: remove console

File: polling-app/src/Components/UpdatePoll/UpdatePoll.js (hunk 1 above)
Comment (imsandip2085) [reply polling-app/src/Components/UpdatePoll/UpdatePoll.js]: ok

File: polling-app/src/Redux/Action/UpdatePoll/addNewOption.js (hunk 2 above)
Comment (imsandip2085) [reply polling-app/src/Redux/Action/UpdatePoll/addNewOption.js]: ok sir

File: polling-app/src/Redux/Action/UpdatePoll/deleteOption.js (hunk 3 above)
Comment (imsandip2085) [reply polling-app/src/Redux/Action/UpdatePoll/deleteOption.js]: ok sir

**Pull request merged**